src/server/cmd/worker/main.go

```diff
@@ -10,6 +10,8 @@ import (
 	"time"
 
 	etcd "github.com/coreos/etcd/clientv3"
+	log "github.com/sirupsen/logrus"
```

Comment:
Am I missing it or is there not a new call to `log` being introduced in this file? I presumably must be, since otherwise it wouldn't compile... but I don't see it. I'm wondering if maybe this got included by `GoImports` for a different `log` call that's not supposed to be using logrus.

Reply:
This was actually just reorganized (because we normally do standard packages, third-party packages, then pachyderm packages).